config/config.exs

```diff
@@ -21,6 +21,9 @@ config :signs_ui,
 # HTTP config
 config :signs_ui, :redirect_http?, false
 
+config :signs_ui,
+  api_key: "dXNlcm6hbWU6cGFzc3dvcmQ="
```

Comment:
Can you make this `"API_KEY"` or something more obviously just a place holder?

Reply:
If this is a real API key, it should be rebased off the branch as well.

Reply:
yeah